src/mon/FSCommands.cc

```diff
@@ -681,6 +681,38 @@ class SetHandler : public FileSystemCommandHandler
         fs->mds_map.set_min_compat_client(vno);
       };
       fsmap.modify_filesystem(fs->fscid, std::move(f));
+    } else if (var == "reject_client_session") {
+      bool reject_session = false;
+      int r = parse_bool(val, &reject_session, ss);
+      if (r != 0) {
+        return r;
+      }
+
+      if (reject_session) {
+        if (!(fs->mds_map.get_reject_client_session())) {
+          fsmap.modify_filesystem(
+            fs->fscid,
+            [n](std::shared_ptr<Filesystem> fs)
+          {
+            fs->mds_map.set_reject_client_session(true);
```

Comment:
Use an mdsmap flag instead of a separate variable. See `CEPH_MDSMAP_ALLOW_STANDBY_REPLAY` and `allow_standby_replay` above this code.

Reply:
Done, PTAL